server/src/leap/soledad/server/auth.py

```diff
@@ -411,6 +412,9 @@ def _verify_token_in_couch(self, uuid, token):
                                       either missing in the tokens db or is
                                       invalid.
         """
+        cache = get_cache_for('authorized', expire=120)
+        if uuid in cache:
```

Comment:
Shouldn't we also validate the token, not only the uuid? Not sure, but wouldn't this currently mean that I could hijack an ongoing sync process by just knowing the uuid?